odm_sdk/utils.py

```diff
@@ -69,6 +71,9 @@ def make_connection_parser(user=None, password=None, host=None, token=None):
                        help='user password')
     group.add_argument('--token', dest='token', default=token, metavar='<api-token>',
                        help='API token to be used instead of the login and password')
+    group.add_argument('--access-token', dest='access_token', default=access_token, metavar='<access-token>',
+                       help='Access token or environment variable with its value to be used instead '
```

Comment:
smth about oauth? Let's double check with product guys

Reply:
asked Mikhail Afanasev

Reply:
@natalyatodorova @m-smazhevsky it seems okay for me